### PR TITLE
Whitespace fixes

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

**Comment by sarahsturgeon** [comment]
retest this please

File: lua/autorun/sh_waiter.lua [hunk 1]
```diff
@@ -42,7 +42,7 @@ end
 local function attendPatron( patronID, patron )
     print( "[CFC Waiter] Attending to patron ID: " .. tostring( patronID ) )
     if patron.attempts >= patron.maxAttempts then
-        print( "[CFC Waiter] Patron ID " .. tostring( patronID ) .. " has reached max attempts! Running onTimeout and removing.." )
+        print( "[CFC Waiter] Patron ID " .. tostring( patronID ) .. " has reached max attempts! Running onTimeout and removing .. " )
```
Comment (overlordbleck) [inline lua/autorun/sh_waiter.lua]:
```suggestion
        print( "[CFC Waiter] Patron ID " .. tostring( patronID ) .. " has reached max attempts! Running onTimeout and removing..." )
```

File: lua/autorun/sh_waiter.lua [hunk 2]
```diff
@@ -70,19 +70,19 @@ local function attendPatrons()
...
+    -- print( "[CFC Waiter] Next delay time: " .. tostring( delayTime ) )
 
     timer.Simple( delayTime, attendPatrons )
 end
 
 local function getPatronsFromQueue()
-    print( "[CFC Waiter] Retrieving patrons from WaiterQueue.." )
-    for _, patron in pairs(WaiterQueue) do
-        print( "[CFC Waiter] Found patron in WaiterQueue! Importing.." )
+    print( "[CFC Waiter] Retrieving patrons from WaiterQueue .. " )
```
Comment (overlordbleck) [inline lua/autorun/sh_waiter.lua]:
```suggestion
    print( "[CFC Waiter] Retrieving patrons from WaiterQueue..." )
```

File: lua/autorun/sh_waiter.lua [hunk 3]
```diff
@@ -70,19 +70,19 @@ local function attendPatrons()
...
+    for _, patron in pairs( WaiterQueue ) do
+        print( "[CFC Waiter] Found patron in WaiterQueue! Importing .. " )
```
Comment (overlordbleck) [inline lua/autorun/sh_waiter.lua]:
```suggestion
        print( "[CFC Waiter] Found patron in WaiterQueue! Importing..." )
```

**Review by overlordbleck — CHANGES_REQUESTED** [review]

File: lua/autorun/sh_waiter.lua (hunk 1 above)
Comment (justinmuskopf) [reply lua/autorun/sh_waiter.lua]: Ah, you got him! 
**HE USED SED, HE DIDN'T ACTUALLY DO ANY LEGWORK**

**Review by justinmuskopf — APPROVED** [review]
Approving based on @OverlordBleck's suggestions being resolved

File: lua/autorun/sh_waiter.lua (hunk 1 above)
Comment (sarahsturgeon) [reply lua/autorun/sh_waiter.lua]: _akshually, bleck_ there were only _two_ dots in the original message..

at least my automated tool can get the number of dots right. bleck can't even seem to.

**Pull request merged**